services/web-app/pages/assets/[host]/[org]/[repo]/[library]/[ref]/index.module.scss

```diff
@@ -32,3 +31,8 @@
     width: 50%;
   }
 }
+
+.resources-heading {
+  margin-top: spacing.$spacing-11;
+  margin-bottom: spacing.$spacing-07;
+}
```

Comment:
```suggestion
```

The H2 component should have this built in already.

Reply:
thanks 🙏